### PR TITLE
Stomach layers

### PR DESCRIPTION
**Pull request by mlin865**

### CODE REVIEW

**Review by rchristie — CHANGES_REQUESTED** [review]
Looks great.
Only one real issue: The Ostium is not working as expect for me (creating it on its own). If I set more than one layer through the wall of the ostium or vessels, it multiplies the wall thickness. Expected behaviour is for the overall thickness to be as specified, but for the relative thicknesses to proportionately distribute layer thicknesses from the overall thickness.
Everything else is mainly questions, some comments in code, plus:
The rat stomach has gaps in the 'mucosa of stomach' between esophagogastric junction and body. Check that is as expected.

File: src/scaffoldmaker/meshtypes/meshtype_3d_colonsegment1.py [hunk 1]
```diff
@@ -74,12 +74,12 @@ def getDefaultOptions(parameterSetName='Default'):
             'Start tenia coli width derivative': 0.0,
             'End tenia coli width': 10.0,
             'End tenia coli width derivative': 0.0,
-            'Tenia coli thickness': 1.6,
+            'Tenia coli thickness': 0.6,
```
[the remark this answers is not in the document]
Comment (mlin865) [reply src/scaffoldmaker/meshtypes/meshtype_3d_colonsegment1.py:77]: Yes the tenia coli thickness has been updated to match the measurements of human colons (recent data from Simon Brookes)

File: tests/test_cecum.py [hunk 2]
```diff
@@ -82,10 +82,11 @@ def test_cecum1(self):
         fieldcache = fieldmodule.createFieldcache()
         result, surfaceArea = surfaceAreaField.evaluateReal(fieldcache, 1)
         self.assertEqual(result, RESULT_OK)
-        self.assertAlmostEqual(surfaceArea, 65959.34314511667, delta=1.0E-6)
+        self.assertAlmostEqual(surfaceArea, 65960.20655074248, delta=1.0E-6)
```
[the remark this answers is not in the document]
Comment (mlin865) [reply tests/test_cecum.py:85]: Noted.

**Comment by mlin865** [comment]
I have fixed the issue with ostium and I confirm there is a gap in the mucosa around the esophagus for rat stomach.

**Review by rchristie — APPROVED** [review]

**Pull request merged**